web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -1536,25 +1457,126 @@ tr.draggable td:nth-child(2) {
   }
 
   .paragraph-type-top {
+    margin-bottom: 15px;
     float: right;
   }
 
   th.field-label h4 {
     margin: 0;
+    background-color: #FFFFFF;
+
+    &:hover {
+      background-color: #FFFFFF;
+    }
+  }
+
+  th.field-label {
+    padding-left: 0px;
   }
 
   table {
-    border-top: 1px solid #e0e0e0;
-    border-bottom: 1px solid #e0e0e0;
+    border-top: 1px solid #FF904B;
+  }
+}
+
+//BP - Changes to Edit Profile page //
+
+.field--widget-entity-reference-paragraphs {
+  position: relative;
+  .paragraphs-dropbutton-wrapper {
+    position: absolute;
+    right: 0;
+    top: 10px;
+  }
+
+  tr {
+    div.form-wrapper {
+      border-top: none;
+      border-bottom: none;
+
+      .field--type-email, .field--type-telephone {
+        margin-right: 100px;
+      }
+    }  
+
+    td div.form-wrapper {
+      padding-top: 0;
+
+      .paragraph-type-top {
+        float: right;
+        margin-top: 25px;
+        z-index: 30;
+
+        .paragraphs-dropbutton-wrapper {
+          position: relative;
+          right: 0;
+          top: 0;
+        }
+      }
+    }
+  }
+
+  div.form-wrapper {
+    border-top: 1px solid #FF904B;
+    border-bottom: 1px solid #FF904B;
+    padding-top: 10px;
+    padding-bottom: 15px;
+    color: #FF904B;
+    margin-bottom: 20px;
+    position: relative;
+
+    strong {
+      font-weight: bold;
+      font-size: 20px;
+    }
+
+    div.form-wrapper {
+      border-top: none;
+      border-bottom: none;
+      padding-top: 0;
+      padding-bottom: 0;
+      margin-bottom: 0;
+      z-index: 20;
+
+      em {
+        color: #000000;
+      }
+    }
   }
 }
 
+a.btn.pull-right {
+  background-color: #1E5168;
+  color: #FFFFFF;
+  margin-top: 13px;
+  margin-right: 25px;
+}
+
+strong {
+  font-size: 15px;
+}
+
+a.use-ajax {
```

Comment:
.use-ajax probably isn't the best class to use? Is there another class .button.

Reply:
Resolved to delete rule